### PR TITLE
Ser 228 limit nutrition training

### PR DESCRIPTION
**Pull request by Bahatiroben**
https://telerivet.com/p/0c6396c9/service/SV7684c662f118f82e/edit

## code review:
- Have renewed the trainings menu, to adopt the new options easily
- have restricted the the nutrition trainings on some sites
 ____________________________

## TESTING
- dial in the shortcode
- enter the account number: 13323509
(this account number belongs to site **Matungu**) which is on the list of sites that are not allowed to access the nutrition training (https://telerivet.com/p/0c6396c9/data/Restricted_20sites_20from_20nutrition_20training?sort=)

- choose option 4 for trainings
- there will be a list of trainings
- with the above account number, there shall not be option for nutrition training which is supposed to be option 12.

On the other hand, when you use account number **26215997** and go through the same process, the option will be available.

### CODE REVIEW

**Review by nyabongo — CHANGES_REQUESTED** [review]
Hey Robben I think you forgot to update the changelog.

**Comment by Bahatiroben** [comment]
> Hey Robben I think you forgot to update the changelog.

Ohhh! I surely have forgot it. but have added it @nyabongo

**Review by nyabongo — CHANGES_REQUESTED** [review]
You seem to have 3 tests failing

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,16 @@ All notable changes to this project will be documented in this file.
 
 ## 2020-10-22
 ### Added
+* [SER-228](https://oneacrefund.atlassian.net/browse/SER-228) Prevent control sites from accessing nutrition training
 
+## 2020-10-22
+### Added
```
Comment (nyabongo) [inline CHANGELOG.md:9]: the date  2020-10-22 and the added section is duplicated here.

Comment (nyabongo) [inline CHANGELOG.md:9]: the date  2020-10-22 and the added section is duplicated here.

**Review by nyabongo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Bahatiroben) [reply CHANGELOG.md:9]: @nyabongo sorry. I missed this. just changed to today's date.

**Review by nyabongo — APPROVED** [review]

**Review by DevFojo — APPROVED** [review]

**Pull request merged**